src/SensioLabs/Melody/Handler/FileHandler.php

```diff
@@ -36,6 +36,11 @@ public function createResource($filename)
             sprintf('file://%s', realpath($filename))
         );
 
-        return new LocalResource($filename, file_get_contents($filename), $metadata);
+        $content = file_get_contents($filename);
+        if (0 !== strpos('#!', $content)) {
```

Comment:
Not sure if I should do this instead :thinking: 
```suggestion
        if ('#!' !== substr($content, 0, 2)) {
```

Reply:
Indeed, I'd rather use `substr`, to avoid to perform a "full string" search when the 2 first bytes are enougth